### DISCUSSION TITLE
DBeaver prompts for 'nag screen' to make exclusions from MS Defender.

### DISCUSSION THREAD

**Discussion by NitiSriAB** [dbeaver/dbeaver#36254] [General]
DBeaver presents the user a ‘nag screen’ on launch of shortcut requesting to make exclusions to Defender to avoid performance degradation. Request you to provide the recommend option to go with.
![dbeaver_prompt](https://github.com/user-attachments/assets/7a517fc6-b21b-4934-9fe2-a77ebc73c012)

**Comment by chirontt** [comment]
It's the Eclipse framework used in DBeaver that presents this 'nag screen'. Just exclude DBeaver from being scanned (the first option) and this 'nag screen' won't appear again in subsequent launches.

**Reply by NitiSriAB** [reply]
Hi Team,
After selecting the first option (excluding DBeaver from being scanned) , it prompts the below error. 
![image](https://github.com/user-attachments/assets/d0614251-cb96-4cb1-8791-e72821807961)
For excluding Dbeaver from being scanned by Microsoft Defender, a relevant document shall be required for our Engineering and Security teams to review .  
Thanks.

**Comment by NitiSriAB** [comment]
Hi Team,

After selecting the first option (excluding DBeaver from being scanned) , it prompts the below error.
***@***.***

For excluding Dbeaver from being scanned by Microsoft Defender, a relevant document shall be required for our Engineering and Security teams to review .

Thanks

**Comment by E1izabeth** [comment]
The issue about this behavior is here #36187
